src/main.go

```diff
@@ -128,11 +138,106 @@ func (app *App) parse(b []byte) ([]byte, error, error) {
 	return pc, errParse, err
 }
 
+func (app *App) parseRemoteURL(urlString string) ([]byte, error, error) {
+	log.Infof("called parseRemoteURL() url: %s", urlString)
+	p := publiccode.NewParser()
+	errParse := p.ParseRemoteFile(urlString)
+	pc, err := p.ToYAML()
+
+	return pc, errParse, err
+}
+
+func promptError(err error, w http.ResponseWriter,
+	httpStatus int, mess string) {
+
+	log.Errorf(mess+": %v", err)
+
+	message := Message{
+		Status:  httpStatus,
+		Message: mess,
+		Error:   err.Error(),
+	}
+	log.Debugf("message: %v", message)
+	w.Header().Set("Content-type", "application/json")
+	w.WriteHeader(message.Status)
+	json.NewEncoder(w).Encode(message)
+}
+
```

Comment:
This two funcs look very similar, I'd try to figure out a way to DRY, what do you think?

Reply:
yes, I completely agree, I will do some hard refactor in a separate PR.